test/dot_s/indidx.s

```diff
@@ -0,0 +1,93 @@
+	.text
+	.file	"cfi-wasm.bs.bc"
+	.type	a,@function
+a:                                      # @a
+	.indidx  	0
```

Comment:
Should this test maybe have these functions in a different order than 0-1-2-3, so that it tests that the indexes aren't accidentally assigned just in order of appearance in the file?